### PR TITLE
To disable Indexed DB

### PR DESCRIPTION
**Pull request by PeterWangIntel**
Reduced size = 230K

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@heke123 @halton 
The pair patch in Blink is 
https://github.com/crosswalk-project/blink-crosswalk/pull/85

File: content/browser/renderer_host/render_process_host_impl.cc [hunk 1]
```diff
@@ -801,11 +801,13 @@ void RenderProcessHostImpl::CreateMessageFilters() {
   AddFilter(new ClipboardMessageFilter);
   AddFilter(new DOMStorageMessageFilter(
       storage_partition_impl_->GetDOMStorageContext()));
+#ifndef DISABLE_INDEXEDDB
   AddFilter(new IndexedDBDispatcherHost(
```
Comment (heke123) [inline content/browser/renderer_host/render_process_host_impl.cc]: do we need also remove codes at line 66,67, 894?

File: content/browser/storage_partition_impl.cc [hunk 2]
```diff
@@ -536,7 +538,11 @@ StoragePartitionImpl* StoragePartitionImpl::Create(
   StoragePartitionImpl* storage_partition = new StoragePartitionImpl(
       context, partition_path, quota_manager.get(), appcache_service.get(),
       filesystem_context.get(), database_tracker.get(),
+#ifndef DISABLE_INDEXEDDB
       dom_storage_context.get(), indexed_db_context.get(),
+#else
+      dom_storage_context.get(), nullptr,
```
Comment (heke123) [inline content/browser/storage_partition_impl.cc]: In BrowserContext::SaveSessionState()
{ ...
    if (indexed_db_context_impl->TaskRunner())    ===> will cause crash here? de-reference a nullptr.
}

**Comment by heke123** [comment]
@PeterWangIntel

**Comment by PeterWangIntel** [comment]
Seems, the ' BrowserContext::SaveSessionState' is for test building, but it's more safe to exclude that. Thank you fro reminding.

**Comment by PeterWangIntel** [comment]
@heke123 ,
New patch is uploaded.

**Comment by heke123** [comment]
LGTM

**Pull request merged**